README.md

```diff
@@ -1,21 +1,51 @@
 # console-game
 Easter egg console.log game in the text based RPG fashion.
 
-## Development Server
-
-Install node.js and run this to install dependencies:
-
-```
-npm install
-```
-
-Then this to start the server:
-
-```
-npm start
-```
-
-Open your browser to `http://localhost:1337'
```

Comment:
I feel like this part of the `README.md` might still be necessary in case someone wants to work on it and add npm dependencies and such.